python/requirements.txt

```diff
@@ -7,5 +7,4 @@ urllib3==1.26.6
 python-pptx==0.6.19
 fastapi==0.75.2
 uvicorn==0.15.0
-psycopg-binary==3.0.11
```

Comment:
Had to revert using psycopg3 because  psycopg-binary 3.0.11 was not found within some virtual environment that azure-functions-core-tools uses. And that was because azure-functions-core-tools uses out-dated pip (version 20.x) that does not have psycopg-binary 3.0.11